### PR TITLE
Depednencies bump up

### PR DESCRIPTION
**Pull request by Scooletz**
Connects to #19

### CODE REVIEW

**Comment by Scooletz** [comment]
Ping @Particular/azure-maintainers

File: GitVersion.yml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 assembly-versioning-scheme: Major
-next-version: 1.0.0
-commit-message-incrementing: Disabled
+next-version: 1.0
 branches:
   develop:
     tag: alpha
+  releases?[/-]:
```
Comment (SeanFeldman) [inline GitVersion.yml]: What's the reason this removed? Just to drop `rc` from the packages built with `release-x` branches?

Comment (Scooletz) [inline GitVersion.yml]: But it's added... before this there was no `release` in this config.

Comment (SeanFeldman) [inline GitVersion.yml]: Not sure what I was smoking 😃

**Pull request merged**